services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -776,6 +776,9 @@ public Response getNewsIllustration(@Context Request rsRequest,
       EntityTag eTag = (size == null || size.isBlank()) ? new EntityTag(String.valueOf(lastUpdated)) : new EntityTag(lastUpdated+"-"+size);
       Response.ResponseBuilder builder = rsRequest.evaluatePreconditions(eTag);
       if (builder == null) {
+        if (news.getIllustrationMimeType().equalsIgnoreCase("image/gif")) {
```

Comment:
```suggestion
        if ("image/gif".equalsIgnoreCase(news.getIllustrationMimeType())) {
```
Just a question, is the image cached correctly by the browser ?